modules/framework/galasa-parent/galasa-boot/src/main/java/dev/galasa/boot/MonitorConfiguration.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright contributors to the Galasa project
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package dev.galasa.boot;
+
+import java.util.List;
+
+public class MonitorConfiguration {
+    
+    private String stream;
+    private List<String> includesRegexList;
+    private List<String> excludesRegexList;
+
+    public MonitorConfiguration(String stream, List<String> includesRegexList, List<String> excludesRegexList) {
+        this.stream = stream;
+        this.includesRegexList = includesRegexList;
```

Comment:
I guess we should call them globPatterns or something if we move away from regexes ?

Reply:
Done in https://github.com/galasa-dev/galasa/pull/236/commits/fa36d081355b9cd4e5ef366e28cde2f8089f0b53